web/portal/user/src/entities/CallForwardSetting/CallForwardSetting.tsx

```diff
@@ -0,0 +1,89 @@
+import AccountTreeIcon from '@mui/icons-material/AccountTree';
+import EntityInterface from '@irontec/ivoz-ui/entities/EntityInterface';
+import _ from '@irontec/ivoz-ui/services/translations/translate';
+import defaultEntityBehavior from '@irontec/ivoz-ui/entities/DefaultEntityBehavior';
+import selectOptions from './SelectOptions';
+import Form from './Form';
+import { foreignKeyGetter } from './ForeignKeyGetter';
+import { CallForwardSettingProperties } from './CallForwardSettingProperties';
+import foreignKeyResolver from './ForeignKeyResolver';
+
+const properties: CallForwardSettingProperties = {
+  callTypeFilter: {
+    label: _('Call type filter'),
+    enum: {
+      internal: _('Internal'),
+      external: _('External'),
+      both: _('Both'),
+    },
+  },
+  callForwardType: {
+    label: _('Call forward type'),
+    enum: {
+      inconditional: _('Inconditional'),
+      noAnswer: _('No Answer'),
+      busy: _('Busy'),
+      userNotRegistered: _('User NotRegistered'),
+    },
+  },
+  targetType: {
+    label: _('Target type'),
+    enum: {
+      number: _('Number'),
+      extension: _('Extension'),
+      voicemail: _('Voicemail'),
+      retail: _('Retail'),
+    },
+  },
+  numberValue: {
+    label: _('Number value'),
+  },
+  noAnswerTimeout: {
+    label: _('No answer timeout'),
+  },
+  enabled: {
+    label: _('Enabled'),
+  },
+  id: {
+    label: _('Id'),
+  },
+  user: {
+    label: _('User'),
+  },
+  extension: {
+    label: _('Extension'),
+  },
+  voicemail: {
+    label: _('Voicemail'),
+  },
+  numberCountry: {
+    label: _('Number country'),
+  },
+};
+
+const columns = [
+  'enabled',
+  'callTypeFilter',
+  'callForwardType',
+  'targetType',
+  'extension',
+];
+
+const CallForwardSetting: EntityInterface = {
+  ...defaultEntityBehavior,
+  icon: AccountTreeIcon,
+  iden: 'CallForwardSetting',
+  title: _('Call forward setting', { count: 2 }),
+  path: '/my/call_forward_settings',
+  toStr: (row: any) => row.id,
+  properties,
+  columns,
+  selectOptions: (props, customProps) => {
```

Comment:
`selectOptions,`
Wrapping selectOptions was required some time ago by ivoz-ui but it's not anymore